src/scss/_components.hero.scss

```diff
@@ -274,7 +274,7 @@ $z-heading: 600;
     opacity: 0.18;
     mix-blend-mode: screen;
 
-    &::before {
+    .cmp-hero--animated &::before {
```

Comment:
I'm not seeing this class animated anywhere, which is resulting in the shapes not showing on any browsers. I think we need to initiate the observer in `hero.js` for this to work.